src/pages/functions/getEvents.ts

```diff
@@ -0,0 +1,15 @@
+export default function getEvents(orgId : string | string[]) {
+    return async () : Promise<{ id: number, title: string }[]> => {
```

Comment:
What is the return type of this function and why?

Reply:
It's returning a promise of an array of objects consisting of number and string values with keys `id` and `title`. Not entirely sure about this solution, but it was the only one I found working.

Reply:
Correct! Put in other words:

* `getEvents()` returns an anonymous (unnamed) function
* That function should return a promise
* That promise should resolve to an array
* That array should contain objects with keys `id` (number) and `title` (string)

So my follow-up question would be: What are the objects containing `id` and `title`? They are javascript objects of course, but what do they represent?

Reply:
Thanks! That is a more clear description.

They represent each event to be rendered in the OrgEventsPage component, through `useQuery` via `queryClient`. But maybe you have something else in mind?

Reply:
Again, you are entirely correct!

So my next question would be: do events just contain an `id` and a `title`, or are there more fields on an event object that are relevant, or will be relevant in the near future?